### ISSUE TITLE
IID Validation Set

### ISSUE THREAD

**Issue by SnowdenLee** [eccv22-ood-workshop/ROBIN-dataset#14]
Hi, 
will you also release the IID validation set? Currently, only training set and OoD validation set are provided.
Thanks a lot!

**Comment by DTennant** [comment]
Hi,
The IID set is released, you can checkout the README of this repo, I have updated the link.
